eth/client.go

```diff
@@ -713,16 +714,28 @@ func (c *client) RegisteredTranscoders() ([]*lpTypes.Transcoder, error) {
 func (c *client) IsAssignedTranscoder(jobID *big.Int) (bool, error) {
 	jInfo, err := c.JobsManagerSession.GetJob(jobID)
 	if err != nil {
+		glog.Errorf("Error getting job: %v", err)
 		return false, err
 	}
 
-	blk, err := c.backend.BlockByNumber(context.Background(), jInfo.CreationBlock)
-	if err != nil {
+	var blk *types.Block
+	getBlock := func() error {
+		blk, err = c.backend.BlockByNumber(context.Background(), jInfo.CreationBlock)
+		if err != nil {
+			glog.Errorf("Error getting block by number %v: %v. retrying...", jInfo.CreationBlock.String(), err)
+			return err
+		}
+
+		return nil
+	}
+	if err := backoff.Retry(getBlock, backoff.WithMaxRetries(backoff.NewConstantBackOff(time.Second), SubscribeRetry)); err != nil {
```

Comment:
Why use a constant backoff here, but an exponential backoff strategy in `CreateTranscodeJob` in `livepeernode.go`?

Reply:
I think I removed all the exponential backoffs.